source/_build/html/.buildinfo

```diff
@@ -0,0 +1,4 @@
+# Sphinx build info version 1
```

Comment:
Looks like you accidently committed the `_build` directory. Please revert this.

Reply:
Done this as well. Excluded it from my latest commit.